root/utility/tableColumns.js

```diff
@@ -239,9 +239,12 @@ export function defineInstrumentUsageColumn(
 
 export function defineNameColumn<T: CoreEntityT | CollectionT>(
   title: string,
+  {descriptive = true, showCaaPresence = false}: {
+    descriptive?: boolean,
+    showCaaPresence?: boolean,
+  },
```

Comment:
IMHO, all parameters to `define*Column` should be named, just like for React components already.
That would also avoid adding meaningless empty `{}` all over the code.

Reply:
@reosarevok when you have time, you can update this PR now that the named parameter stuff got merged. should resolve this comment